PC02_Graffiti_Kanu.py

```diff
@@ -0,0 +1,118 @@
+#!/usr/bin/env python
+# coding: utf-8
+
+'''
+Turtle starter code
+ATLS 1300
+Author: Dr. Z
+Author: Isha Kanu
+September 09, 2020
+'''
+
```

Comment:
Adding code description:
```suggestion

"""
Graffiti artwork piece of Jeff Bezos using the Turle library. The artwork is a warning of Jeff Bezos's devil-ish money stealing ways.
"""

```